db/migrate/20181008201815_update_instagram_data.rb

```diff
@@ -0,0 +1,9 @@
+class UpdateInstagramData < ActiveRecord::Migration
+  def change
+    enterprises = Enterprise.where("instagram like ?", "%instagram.com%")
+    enterprises.each do |e|
```

Comment:
We should rather use [find_each](https://apidock.com/rails/v3.2.13/ActiveRecord/Batches/find_each) here IMO. This can potentially be a large list of enterprises.

Reply:
Ok, I agree with it. I will make this change.